Formula/tor.rb

```diff
@@ -39,15 +39,6 @@ def install
     system "make", "install"
   end
 
-  def caveats; <<-EOS.undent
```

Comment:
This is not Homebrew specific information. Homebrew specific information is what the caveats are for so 👍 to removing them.